src/store/actions.ts

```diff
@@ -1,17 +1,17 @@
 import { ActionContext } from 'vuex';
 import RootState from './RootState';
-import SearchEntityRepository from '@/data-access/SearchEntityRepository';
 import SearchResult from '@/data-access/SearchResult';
 import Error from '@/data-model/Error';
 import Property from '@/data-model/Property';
 import PropertyValueRelation from '@/data-model/PropertyValueRelation';
+import QueryBuilderServices from '@/QueryBuilderServices';
 
 // eslint-disable-next-line max-len
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type, @typescript-eslint/explicit-module-boundary-types
-export default ( searchEntityRepository: SearchEntityRepository ) => ( {
+export default ( services: QueryBuilderServices ) => ( {
```

Comment:
No, this replaces the dependency injection with the service locator anti-pattern. Rather, this should inject both services individually here.

Currently, this may feel a bit pointless, but it will make more sense as soon as we use more services in other places as well.

(However, I think it is fine as it is in App.vue)

Reply:
hmm, makes sense. Fixed now.